web/src/components/assets/AssetCard.tsx

```diff
@@ -0,0 +1,88 @@
+import { Asset } from "@/openapi";
+import styles from "./AssetCard.module.css";
+
+type AssetCardProps = {
+  asset: Asset;
+};
+
+const formatClassification = (classification: string) => {
+  return classification.replace(/_/g, " ");
+};
+
+const formatProtection = (protection: string) => {
+  return protection.replace(/_/g, " ");
+};
+
+const formatDate = (dateString: string) => {
+  try {
+    return new Date(dateString).toLocaleDateString("en-GB", {
+      year: "numeric",
+      month: "short",
+      day: "numeric",
+      hour: "2-digit",
+      minute: "2-digit",
+    });
+  } catch {
+    return dateString;
+  }
+};
+
+export default function AssetCard({ asset }: AssetCardProps) {
+  const getClassificationClass = (classification: string) => {
+    switch (classification) {
+      case "public":
+        return styles["asset-classification-public"];
+      case "confidential":
+        return styles["asset-classification-confidential"];
+      case "highly_confidential":
+        return styles["asset-classification-highly-confidential"];
+      default:
+        return styles["asset-classification-public"];
+    }
+  };
+
+  return (
+    <div className={styles["asset-card"]}>
+      <div className={styles["asset-header"]}>
+        <h4 className={styles["asset-title"]}>{asset.title}</h4>
+        <div className={styles["asset-meta"]}>
+          <span className={`${styles["asset-badge"]} ${getClassificationClass(asset.classification_impact)}`}>
+            {formatClassification(asset.classification_impact)}
+          </span>
```

Comment:
might be nice pulled right

Reply:
Hmm, not quite sure what you are referring to there. They are pulled to the right of the card on my screen.

<img width="604" height="215" alt="Screenshot 2025-08-29 at 16 46 59" src="https://github.com/user-attachments/assets/5e0d3df8-5ee9-41af-b496-63ce71344da9" />